src/content/doc-surrealdb/deployment/surreal cloud.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 1
+sidebar_label: Deploy on Surreal Cloud (Beta)
+title: Deploy on Surreal Cloud (Beta) | Deployment
+description: This tutorial guides users through the process of deploying SurrealDB using Surreal Cloud.
+---
+
+import Image from "@components/Image.astro";
+
+import NamespaceDatabaseLight from '@img/image/cloud/namespace-database.png';
+import NamespaceDatabaseDark from '@img/image/cloud/namespace-database.png';
+
+import LightCloud from '@img/image/cloud/surrealcloud.png';
+import DarkCloud from '@img/image/cloud/surrealcloud.png';
+
+import OpenInSDK from '@img/image/cloud/open-in-sdk.png';
+
+import QueryingInstanceLight from '@img/image/cloud/querying-instance.png';
+import QueryingInstanceDark from '@img/image/cloud/querying-instance.png';
+
+import OpenInHTTPCurlLight from '@img/image/cloud/open-in-http.png';
+import OpenInHTTPCurlDark from '@img/image/cloud/open-in-http.png';
+
+<Image
+alt="Surreal Cloud (Beta)"
+width={100}
+className="flag-image-title"
+src={{
+	light: LightCloud,
+	dark: DarkCloud,
+	}}  
+/>
+# Deploy on Surreal Cloud (Beta)
+
+## What is Surreal Cloud (Beta)?
+
+Surreal Cloud is a fully managed database service that provides a seamless deployment experience for SurrealDB. It's designed to be lightweight, leveraging the Open-Source SurrealDB product as much as possible. Its features are intended to complement those of the open-source version and offer you a seamless deployment experience.
+
+In this guide, we'll walk through the process and options to deploy SurrealDB using [Surreal Cloud](/docs/cloud). 
+
+## Requirements
+
+To get [started with Surreal Cloud](/docs/cloud/getting-started), you'll need to have an account. If you don't have one, you can sign up for a free account by [downloading Surrealist](https://github.com/surrealdb/surrealist/releases) or using the [Surrealist web app](https://surrealist.app/cloud). 
+
+
+## Create an Instance
+
+To create an Instance, in Surrealist, click on the **Surreal Cloud** tab and select **Continue**. This will prompt you to the authentication page. If you don't have an account, you'll be asked to [sign up](/docs/cloud/getting-started/create-an-account) with an email address, Google, or GitHub.
+
+After you've signed up, you'll be redirected to the Surreal Cloud dashboard. Here, you can create an Instance by clicking on the **Create Instance** button. 
+
+### Configure your Instance
+
+While creating an Instance, you'll be asked to provide a few details about your Instance. These include:
+
+- **Instance Name**: This is the name of your Instance. It's used to identify your Instance in the Surreal Cloud dashboard.
+- **SurrealDB Version**: This is the version of SurrealDB that you want to use. You can choose the version that best fits your needs.
+- **Instance Region**: This is the region where your Instance will be deployed. You can choose the region that's closest to you for better performance.
```

Comment:
How about "better performance through a lower two-way trip time"? Technically it's latency but some people might be unfamiliar with the word